app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,14 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.github.palFinderTeam.palfinder">
 
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```

Comment:
Are all these lines necessary?

Reply:
i didn't saw the permission got multiplied